### PR TITLE
refactor(playground): add checkbox echo

### PR DESCRIPTION
**Pull request by adnasa**
#### Summary

- adds checkbox to include param for `echo-server`

#### Description

image

### CODE REVIEW

File: playground/src/components/echo-server/echo-server.js [hunk 1]
```diff
@@ -30,12 +31,24 @@ const EchoServer = () => {
...
+    setShouldIncludeParamInRequest,
+  ] = useState(false);
+
+  const onChangeShouldIncludeParamInRequest = (event) => {
+    const nextValue = event.target.value === 'false' ? true : false;
+    setShouldIncludeParamInRequest(nextValue);
+  };
+
   const handleSendRequest = useCallback(() => {
+    const searchParam = shouldIncludeParamInRequest ? `?query=value` : null;
```
Comment (emmenko) [inline playground/src/components/echo-server/echo-server.js:45]: The `?` is already assigned when joining the values below.

```suggestion
    const searchParam = shouldIncludeParamInRequest ? `hello=world` : null;
```

File: playground/src/components/echo-server/echo-server.js [hunk 2]
```diff
@@ -30,12 +31,24 @@ const EchoServer = () => {
...
+    setShouldIncludeParamInRequest,
+  ] = useState(false);
+
+  const onChangeShouldIncludeParamInRequest = (event) => {
+    const nextValue = event.target.value === 'false' ? true : false;
+    setShouldIncludeParamInRequest(nextValue);
+  };
+
   const handleSendRequest = useCallback(() => {
+    const searchParam = shouldIncludeParamInRequest ? `?hello=world` : null;
```
Comment (emmenko) [inline playground/src/components/echo-server/echo-server.js:45]:
```suggestion
    const searchParam = shouldIncludeParamInRequest ? `hello=world` : null;
```

File: playground/package.json [hunk 3]
```diff
@@ -25,12 +25,12 @@
     "@commercetools-frontend/l10n": "18.6.0",
     "@commercetools-frontend/permissions": "18.6.0",
     "@commercetools-frontend/sdk": "18.6.0",
+    "@commercetools-uikit/checkbox-input": "^11.2.0",
```
Comment (adnasa) [inline playground/package.json:28]: haha I just did this, pushed and got rejected. you beat me to it 😓

**Review by emmenko — APPROVED** [review]

File: playground/package.json (hunk 3 above)
Comment (emmenko) [reply playground/package.json:28]: Yeah I was about to write again but I thought to be easier to just push the fix.

**Pull request merged**